README.md

```diff
@@ -0,0 +1 @@
+# graasp-template-backend
```

Comment:
Change name.

Reply:
And do you think you could write some documentation/instructions about stripe here?